src/SixLabors.Fonts/TextOptions.cs

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Collections.Generic;
+using System.Numerics;
+
+namespace SixLabors.Fonts
+{
+    /// <summary>
+    /// Provides configuration options for rendering text.
+    /// </summary>
+    public sealed class TextOptions
+    {
+        private float tabWidth = 4F;
+        private float dpi = 72F;
+        private float lineSpacing = 1F;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextOptions"/> class.
+        /// </summary>
+        /// <param name="font">The font.</param>
+        public TextOptions(Font font) => this.Font = font;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TextOptions"/> class from properties
+        /// copied from the given instance.
+        /// </summary>
+        /// <param name="options">The options whose properties are copied into this instance.</param>
+        public TextOptions(TextOptions options)
+        {
+            this.Font = options.Font;
+            foreach (FontFamily family in options.FallbackFontFamilies)
+            {
+                this.FallbackFontFamilies.Add(family);
+            }
+
+            this.TabWidth = options.TabWidth;
+            this.ApplyHinting = options.ApplyHinting;
+            this.Dpi = options.Dpi;
+            this.LineSpacing = options.LineSpacing;
+            this.Origin = options.Origin;
+            this.WrappingLength = options.WrappingLength;
+            this.WordBreaking = options.WordBreaking;
+            this.TextDirection = options.TextDirection;
+            this.TextAlignment = options.TextAlignment;
+            this.HorizontalAlignment = options.HorizontalAlignment;
+            this.VerticalAlignment = options.VerticalAlignment;
+            this.LayoutMode = options.LayoutMode;
+            this.KerningMode = options.KerningMode;
+            this.ColorFontSupport = options.ColorFontSupport;
+        }
+
+        /// <summary>
+        /// Gets or sets the font.
+        /// </summary>
+        public Font Font { get; set; }
```

Comment:
A project consuming Fonts without `<Nullable>enable</Nullable>` can actually set this to null, so adding guards is still necessary if we assume in Fonts and Drawing code that it's never null.

Same concern around constructor.

Reply:
Good point, Will update.

Reply:
Done